pyproject.toml

```diff
@@ -182,7 +182,7 @@ exclude_dirs = [
 ]
 
 [tool.cibuildwheel]
-build = ["cp27-*", "cp35-*", "cp36-*", "cp37-*", "cp38-*", "cp39-*", "cp310-*", "cp311-*"]
+build = ["cp37-*", "cp38-*", "cp39-*", "cp310-*", "cp311-*", "cp312-*"]
```

Comment:
😮 this probably deserves its own PR

Reply:
oh, probably, this is all ignored anyways since we configure cibuildwheel via env variables, I was surprised that this config is out of date and everything was still working... I wonder if we just drop this 🤷🏻